PR for Carousel Effect

shubham-sharma-2323 opened the pull request:
Here's the pull request of task #656 of adding a carousel effect using css

![Screenshot (124)](https://user-images.githubusercontent.com/85223141/169367751-f40ba3ee-4b88-4cc1-8acf-b7d2e058199f.png)

VidhiBhatt01 commented on Carousel Effect/index.html:
```diff
@@ -0,0 +1,61 @@
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Add a suitable title

VidhiBhatt01 reviewed (changes requested):
Can you add some high-definition images? The current ones are getting distorted. Also add it in style_links.json

https://deploy-preview-689--amazing-css-effects.netlify.app/carousel%20effect/

shubham-sharma-2323 commented:
@VidhiBhatt01 i have changed the title of page, updated style_links.json and added a bit more clear photos (hope so it works). You can check the changes.
